src/content/posts/manage-fork-repository-locally-and-remotely/index.md

```diff
@@ -1,3 +1,12 @@
+---
+title: "Manage fork repository locally and remotely"
+summary: "TODO"
+date: 2024-04-16T20:00:00-02:00
+lastUpdate: 2024-06-03T23:04:17+0200
+tags: ["git", "github"]
+author: Zyno
```

Comment:
```suggestion 
author: Zyno and ctmbl
```
Can we put multiple authors?

Reply:
nop unfortunately 😕 
I have a side project of enablinbg that feature but it's really not a priority and isn't often used
just keep yourself as the author but you might add something like https://github.com/iScsc/blog.iscsc.fr/blob/8e8ad67fe217bac9f1565e192db98b946041ac62/src/content/posts/docker_bind_mounts_introduction.md?plain=1#L11

Reply:
Ok then we will modify it later when it will be merged!
Untill then I will add this line.